### ISSUE TITLE
dei: url with or without schema path?

### ISSUE THREAD

**Issue by michaelwood** [ThreeSixtyGiving/extensions-registry#1]
Currently the extensionBaseUrl doesn't include the `/schema` directory in the path which causes a `404 file not found`

```diff
-- "extensionBaseUrl": "https://raw.githubusercontent.com/ThreeSixtyGiving/360-dei/main/",
++ "extensionBaseUrl": "https://raw.githubusercontent.com/ThreeSixtyGiving/360-dei/main/schema/",
```
Are tools expected to append `/schema` ? or does this just need amending?

**Comment by mrshll1001** [comment]
From `README.md`:

> 4. For each entry in the `schemas` array perform the following:
>  * Construct the URL to the extension schema file with the following pattern: `extensionBaseUrl` + `/schema/` + `extensionSchema`.
>  * Retrieve the file stored at this URL. Create appropriate error messages in the event of HTTP 40x and 50x error codes.
>  * Perform a JSON Schema merge patch of the extension schema file onto either the 360Giving Schema or the 360Giving package schema as indicated by the value of `target`.

The currently expected behaviour is for tools to append the `/schema` however we can change this if it makes more sense to provide direct links to schemas in the schemas array. That would be less work for tools so I'm happy to make that change.

**Comment by mrshll1001** [comment]
I've changed this behaviour in the latest commit to main (40af4782de7fe6b7d730413c06163525fc8ed3e5).

* `schemas/extensionSchema` is now `schemas/uri` and must have the format of URI. It will contain a URI to a full schemafile.
* `codelists` is now an array of URI, and will contain a URI to a full schemafile

I've also updated the DEI registry entry in its branch. Be aware that the URIs in the registry entry are to the main branch of the DEI Extension repo which currently doesn't contain the appropriate version of the DEI Extension. I'll see about merging that in ASAP.
